### DISCUSSION TITLE
Windows deployment error loading kernel too old

### DISCUSSION THREAD

**Discussion by badacook** [cobbler/cobbler#3808] [Q&A]
I have installed cobbler-3.3.6-1  on Fedora 40 Server Edition as deployment platform ,and  tested Fedora35 automated installation Successfully,
so i  follow the guide https://github.com/cobbler/cobbler/blob/v3.3.6/docs/user-guide/wingen.rst to test windwos 11 installation by UEFI iPXE method,
All steps in the guide are normal without adding  extra drivers to the image. the test terminate on boot menu win11-x86_64 when loading too old kernel 
Below is a screenshot of the process ,any help will be great thanks.
[cobblersync.log](https://github.com/user-attachments/files/16593835/cobblersync.log)
![error](https://github.com/user-attachments/assets/7b124d4b-87a4-440b-bf7c-15940a8db629)
![images](https://github.com/user-attachments/assets/fd0358d3-c37e-4e23-979b-06ca057df996)
![menu](https://github.com/user-attachments/assets/89091411-24d4-40d6-9d64-ce36cd9ed78e)
![systeminfo](https://github.com/user-attachments/assets/51443428-828c-4692-a028-ffe4f39da310)

**Comment by sgnay** [comment]
show you `dhcpd.conf` , you should booting from ipxe，but now it's grub

**Reply by badacook** [reply]
thanks for your reply i have solve this problem in the newest cobbler version.
